blocks/footer/footer.css

```diff
@@ -4,6 +4,7 @@ footer {
   background-color: var(--footer-background-color);
   font-size: var(--footer-font-size-copyright);
   color: var(--text-color-on-black-background);
+  clear: both;
```

Comment:
what does this do?

Reply:
Prevents floating images to enter this area. 
https://developer.mozilla.org/en-US/docs/Web/CSS/clear

Reply:
without clear:
<img width="1144" alt="Screenshot 2023-09-11 at 09 30 49" src="https://github.com/hlxsites/24life/assets/2250964/a40c9cca-5cb0-464a-b9b1-c5135dc642f3">

Reply:
Thanks for the image 👍🏽